docs/src/modules/components/ApiPage.js

```diff
@@ -60,6 +60,18 @@ ClassesTable.propTypes = {
 function SlotsTable(props) {
   const { componentSlots, slotDescriptions } = props;
   const t = useTranslate();
+  const componentSlotsWithRootAtFront = componentSlots.reduce((acc, curr) => {
+    if (curr.name === 'root') {
+      acc.unshift(curr);
+    } else {
+      acc.push(curr);
+    }
+    return acc;
+  }, []);
+  const componentSlotsSorted = [
+    componentSlotsWithRootAtFront[0],
+    ...componentSlotsWithRootAtFront.slice(1).sort((a, b) => a.name - b.name),
+  ];
```

Comment:
I think that having this logic inside `ApiPage.js` contradict the motivation to have API `.json` files: so we can audit in git the changes to the API pages.

Reply:
That makes sense. Thanks. I will close this PR as we can go with yours.